packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewer.js

```diff
@@ -72,6 +73,7 @@ const SubjectGroupViewer = forwardRef(function SubjectGroupViewer(props, ref) {
                 
                 image={image}
                 index={index}
+                subjectId={subjectIds[index]}
```

Comment:
I'm not seeing updated tests for this. Should it be tested?

Reply:
I've added tests to 'SGVGridCell' to not only test for SubjectIDs, but for SubjectIDs+cellIndex within the context of annotations.

Reply:
I've also added (lighter) tests to the SubjectGroupViewer to ensure it's rendering those SGVGridCells properly 👍